sdcm/utils/git.py

```diff
@@ -25,6 +30,30 @@ def get_git_current_branch() -> str:
     return proc.stdout.decode(encoding="utf-8").strip()
 
 
+def get_git_status_info() -> GitStatus:
+    try:
+        status_proc = subprocess.run(args=["git", "status", "-b", "--porcelain=v2",
+                                     ".nothing"], check=True, capture_output=True)
+        output = status_proc.stdout.decode(encoding="utf-8").strip()
+        git_status: GitStatus = {}
+        for line in [line for line in output.split("\n") if line.startswith("#")]:
+            key, value = line[2:].split(" ", maxsplit=1)
+            git_status[key] = value
+
+        remote, _ = git_status["branch.upstream"].split("/", maxsplit=1)
```

Comment:
```
$ git status -b --porcelain=v2 .nothing
# branch.oid ef4138bacf727f25cadb3267bf487bb66fc7b086
# branch.head dev
```

So, in my case I don't have the `branch.upstream`. What do I do wrong?

Reply:
This should mean that your branch `dev` isn't set to track any of the remote branches.

Reply:
It is refering to the remote branch `dev`:
```
commit ef4138bacf727f25cadb3267bf487bb66fc7b086 (HEAD -> dev, vpssh/dev)
```

Reply:
Seems to be working on my end:
```
git status -b --porcelain=v2 .nothing
# branch.oid dcbdf33eb73024dda3dfc17344e39fa595ac38fa
# branch.head add-argus-branch-tracking
# branch.upstream temichus/master
# branch.ab +692 -0
```